torch/lib/THD/base/Cuda.cpp

```diff
@@ -0,0 +1,23 @@
+#include "Cuda.hpp"
+
+THCState** _THDCudaState;
```

Comment:
It's a bit ugly because now I need to insert code to a CUDA module that will initialize it. But I can do it, if you really don't like it